BitmarkRegistry/Controllers/Main/Properties/MusicBitmarkDetailViewController.swift

```diff
@@ -0,0 +1,333 @@
+ //
+//  MusicBitmarkDetailViewController.swift
+//  BitmarkRegistry
+//
+//  Created by Macintosh on 8/14/19.
+//  Copyright © 2019 thuyentruong. All rights reserved.
+//
+
+import UIKit
+import SnapKit
+import WebKit
+import RxSwift
+import Alamofire
+import RxFlow
+import RxCocoa
+
+class MusicBitmarkDetailViewController: UIViewController, Stepper {
+  var steps = PublishRelay<Step>()
+
+  // MARK: - Properties
+  var bitmarkR: BitmarkR!
+  var assetR: AssetR!
+  lazy var assetFileService = {
+    return AssetFileService(owner: Global.currentAccount!, assetId: assetR.id)
+  }()
+  var waitingScreen: UIView!
+  var activityIndicator: UIActivityIndicatorView!
+  var closeButton: UIButton!
+  var musicWebView: WKWebView!
+  var bitmarkDetailView: UIView!
+  var bitmarkDetailViewTapGesture: UITapGestureRecognizer!
+  var viewBitmarkOptionsBtn: UIButton!
+  var authenticatingTransferBtn: UIButton!
+  var bitmarkDetailViewHeightConstraint: Constraint!
+  var viewBitmarkOptionsHeightConstraint: Constraint!
+  var authenticatingTransferBtnHeightConstraint: Constraint!
+  let disposeBag = DisposeBag()
+  var networkReachabilityManager = NetworkReachabilityManager()
+
+  // MARK: - Init
+  override func viewDidLoad() {
+    super.viewDidLoad()
+
+    navigationItem.backBarButtonItem = UIBarButtonItem()
+    setupViews()
+    setupEvents()
+    loadData()
+    markRead()
+  }
+
+  override func viewWillAppear(_ animated: Bool) {
+    super.viewWillAppear(animated)
+    navigationController?.isNavigationBarHidden = true
```

Comment:
We can use [hidesBottomBarWhenPushed](https://developer.apple.com/documentation/uikit/uiviewcontroller/1621863-hidesbottombarwhenpushed)

Reply:
thanks, but are they different thing? 
NavigationBar is bar in the top. (NavigationBar)
BottomBar is bar in the bottom (TabBar)

Reply:
ok